src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ CHANGELOG
  * Deprecated auto-injection of the container in AbstractController instances, register them as service subscribers instead
  * Deprecated processing of services tagged `security.expression_language_provider` in favor of a new `AddExpressionLanguageProvidersPass` in SecurityBundle.
  * Enabled autoconfiguration for `Psr\Log\LoggerAwareInterface`
+ * Deprecated `Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\CacheCollectorPass`. Use `Symfony\Component\Cache\DependencyInjection\CacheCollectorPass` instead.
```

Comment:
I'd suggest to remove the namespace in the first part, since this changelog is already in "FrameworkBundle"